routes/web.php

```diff
@@ -41,10 +44,11 @@
 Auth::routes(['register' => false]);
 Route::get('/categorie/{category:slug}', 'CategoryController@index')->name('categories.index');
 
+Route::get('/region/{region:slug}', 'PublicController@indexRegional')->name('public.index-region');
 Route::get('/region/province', 'PublicController@indexProvincial')->name('public.index-provincial');
 Route::get('/region/{region:slug}', 'PublicController@indexRegional')->name('public.index-region');
 Route::get('/region/{region:slug}/{category}', 'PublicController@indexRegionalCategories')->name('public.index-region-category');
 Route::get('/entreprise/ajout', 'PlaceController@createPublic')->name('places.create-public');
-Route::post('/entreprise/ajout', 'PlaceController@storePublic')->name('places.store-public')->middleware(ProtectAgainstSpam::class);
```

Comment:
Forget to reactivate the middleware.

Reply:
Oops, my bad. I had to remove it so I could naviguate locally.
Will fix this.